### DISCUSSION TITLE
YAML config file for SocketPublisher.server_uri

### DISCUSSION THREAD

**Discussion by Fenhel** [stella-cv/stella_vslam#617] [Q&A]
I attempted to follow the instructions in the “running on Docker -> instructions for SocketViewer” section for macOS. The instructions mention: 'append the SocketPublisher.server_uri entry to the YAML config file of stella_vslam,' but I can't find the appropriate file. Where should I add this line? https://stella-cv.readthedocs.io/en/latest/docker.html#on-macos

**Comment by Skwangles** [comment]
https://github.com/stella-cv/stella_vslam/tree/main/example%2Faist

You would put this alongside the stella_vslam_examples script you want to use.
You may need to bind the directory you have put it in so Stella can find it from the container.
